### PR TITLE
new fixture for issue 63

### LINKED ISSUE #63: Make test for comp_file format for bad_008?
Acceptance criteria:
- [x] A fixture is created with a leader (positions 6 and 7 should be `ma`) and an 008 field
- [x] Introduce an error in the 008 specific to the format
- [x] A test is created that verifies the `bad_008?` method finds the invalid 008

See https://www.loc.gov/marc/bibliographic/bdleader.html
https://www.loc.gov/marc/bibliographic/bd008.html

### PR DESCRIPTION
**Pull request by Beck-Davis**
Closes #63.

Co-authored by Peter Green <pmgreen@users.noreply.github.com>

### CODE REVIEW

**Comment by mzelesky** [comment]
I would like to have the test included in this pull request along with the fixture. So you can make additional commits within the same branch to write the test to accompany the fixture.

File: spec/fixtures/fixed_fields/marc_with_bad_008.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<collection>
+    <record>
+        <leader>01104nma a2200289 i 4500</leader>
+        <controlfield tag="001">99127156537506421</controlfield>
+        <controlfield tag="008">230519s1996    njua       k  000 0 eng d</controlfield>
```
Comment (mzelesky) [inline spec/fixtures/fixed_fields/marc_with_bad_008.xml:6]: I would like to have the 008 have a value that can be fixed. In the `fix_comp_008` method, that means that it has to be one of the following positions of the field:

audience (position 22 of `u` or `v` becomes `j`)
government publication indicator (position 28 of 'n' becomes 'o')

**Comment by Beck-Davis** [comment]
The Fixture is updated and test created

File: spec/fixed_fields/comp_format_bad_008_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'nokogiri'
+require 'marc'
+require 'byebug'
+require 'marc_cleanup'
+
+RSpec.describe 'test  008 ' do
```
Comment (mzelesky) [inline spec/fixed_fields/comp_format_bad_008_spec.rb:6]: In the `describe` string, the name of the method you're testing should be there in general. In this case, we are testing the `bad_008?` method.

File: spec/fixed_fields/comp_format_bad_008_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  describe 'bad computer format 008' do
+    let(:marcfile) {"#{Dir.getwd}/spec/fixtures/fixed_fields/marc_with_bad_008.xml"}
+    let(:reader) { MARC::XMLReader.new(marcfile, parser: 'magic') }
+    let(:record_with_bad_008) { reader.first }
+    it 'knows that a record with bad 008 is invalid' do
```
Comment (mzelesky) [inline spec/fixed_fields/comp_format_bad_008_spec.rb:11]: This should specify that it's finding a bad computer 008.

**Review by mzelesky — CHANGES_REQUESTED** [review]
Minor changes needed, but the test is successful!

**Review by mzelesky — APPROVED** [review]
Looks great!

**Pull request merged**